plugins/headless/hu.bme.mit.gamma.headless.server/src/hu/bme/mit/gamma/headless/server/util/FileHandlerUtil.java

```diff
@@ -53,6 +53,13 @@ public static int getPid(String workspace, String projectName) throws IOExceptio
 	}
 
 	public static String getProperty(String propertyName) {
+		String path = null;
+		try {
+			path = new File(".").getCanonicalPath();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
```

Comment:
Delete the TODO part, if you do not aim to do anything with it later

Reply:
Maybe this try-catch block is not necessary here at all. Couldn't this path variable be set right before/after `Properties prop = new Properties();`?